src/App.js

```diff
@@ -1,25 +1,24 @@
-import logo from './logo.svg';
-import './App.css';
+import { BrowserRouter, Routes, Route} from 'react-router-dom';
+import { Header } from './components/Header';
+import { Nav } from './components/Nav'
+import { AllReviews } from './components/AllReviews';
+import './App.css'
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <BrowserRouter >
+    <div className="App" >
+     <Header />
+     <Nav />
+
+     <Routes>
+        <Route path='/' element={<AllReviews />} />
+        <Route path='/reviews' element={<AllReviews />} />
```

Comment:
Do we need two routes that lead to the same place but with different url paths? Could it just be your homepage?